package.json

```diff
@@ -152,7 +152,19 @@
           },
           "default": [],
           "description": "Per-workspace list of module directories for the language server to exclude"
-        }
+        },
+        "terraform-ls.experimentalFeatures": {
+          "scope": "resource",
```

Comment:
Looks great overall, something I don't know well enough yet is the scopes, it's documented here

https://code.visualstudio.com/api/references/contribution-points

is `resource` the one we want?